web/src/SidebarAccount.tsx

```diff
@@ -247,21 +244,11 @@ function SidebarAccount(props: SidebarAccountProps) {
         onSubmit={notifyTiltOfRegistration}
       >
         <input name="token" type="hidden" value={cookies.get("Tilt-Token")} />
-        <MenuContentButtonSignUp type="submit" value="Sign Up via GitHub" />
+        <MenuContentButtonSignUp
+          type="submit"
+          value="Link Local Tilt to Tilt Cloud"
```

Comment:
```suggestion
          value="Link Tilt to Tilt Cloud"
```

Let’s not use “Local” since it’s not elsewhere in the product. And customers I think associate Tilt with the installed binary experience already. We have Git and GitHub (I know there are different orgs behind these products) so Tilt and Tilt Cloud seem to work well I think.

Reply:
Sure! I agree with Han's comment in the ticket about sort of wanting a better/less confusing line of copy for this button, but I like that this is concise and it's 100% better than what we have on master today.